ambersim/__init__.py

```diff
@@ -3,7 +3,7 @@
 from pathlib import Path
 
 # package root
-ROOT = str(Path(__file__).parent.parent)
+ROOT = str(Path(__file__).parent.absolute())
```

Comment:
comment for reviewers: since we moved the `/models` subdirectory into `ambersim`, there is no reason for `ROOT` to point to the parent of the package root anymore. Instead, it actually points to the installed package root, which has `/models` as a child subdirectory.